appveyor.yml

```diff
@@ -4,6 +4,7 @@ init:
 
 # Build script
 build_script:
+  - cmd: powershell -NoProfile -ExecutionPolicy unrestricted -Command .\appveyor.ps1
```

Comment:
I've updated the AppVeyor build image to `Visual Studio 2019` so .Net 3.1 should now be installed by default. So you should be able to drop this command now.

Reply:
Ok I updated script and removed unnecessary script - lets see :)

Reply:
Unfortunatelly appveyor is not able to find .net core 3.1 for testing purposes
![image](https://user-images.githubusercontent.com/1914201/108861829-ebe1a800-75ef-11eb-85da-ac2506b911af.png)

Reply:
Hmm, I was going off [this update](https://www.appveyor.com/updates/) from Appveyor that says `.NET Core 3.1.406` is included in the latest `Visual Studio 2019` image but its hard to argue with that build log.

Reply:
ok I will return back script for now if you don't mind :)